fdbserver/ptxn/TLogPeekCursor.actor.h

```diff
@@ -83,10 +81,10 @@ class PeekCursorBase {
 	// invalidate the existing iterators.
 	Future<bool> remoteMoreAvailable();
 
-	// Gets one mutation
+	// Gets one mutation, the behavior is undefined if the cursor has no message currently
 	const VersionSubsequenceMessage& get() const;
 
-	// Moves to the next mutation, return false if there is no more mutation
+	// Moves to the next mutation, the behavior is undefined if the cursor has no message currently
```

Comment:
What's the precondition to call these two functions? In addition to saying the behavior is undefined, state when it's safe to call these functions.